configs/scripts/program/build.sh

```
@@ -1,125 +1,38 @@
-# parse the JSON array into a bash array
-PROGRAM_LINES="$(
-    printf '%s\n' "${PROGRAMS}" |
-        jq -cer 'if type == "array" and length > 0 then .[] else error("PROGRAMS must be a non-empty JSON array") end'
-)"
-PROGRAM_LIST=()
-while IFS= read -r program; do
-    PROGRAM_LIST+=("${program}")
-done <<EOF
-${PROGRAM_LINES}
-EOF
+PROGRAMS=$(echo ${PROGRAMS} | jq -c '.[]' | sed 's/"//g')
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes in this variable will not be respected.

(SC2090)